.github/workflows/testSlackWorkflow.yml

```diff
@@ -8,10 +8,11 @@ jobs:
     uses: ./.github/workflows/slack-notification.yml
     secrets: 
       SLACK_WEBHOOK_URL: ${{ secrets.VSCODE_SLACK_WEBHOOK }}
+      IDEE_RELEASE_ALERT_SLACK_WEBHOOK: ${{ secrets.IDEE_RELEASE_ALERT_SLACK_WEBHOOK }}
     with:
       title: "Test: Release Branch v56.7.1 (patch) successfully created"
       failedEvent: "Create Release Branch"
       successfulEvent: "https://github.com/forcedotcom/salesforcedx-vscode/tree/release/v56.7.1"
       type: "created"
-      result: "Success"
+      result: "failure"
```

Comment:
updated the test action to send a failure.

Reply:
We need to get rid of this before merging right?

Reply:
nah it doesn't hurt anything to be in the repo briefly.  I'll delete it after we verify the flow 👍